keccak-hash/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 
 [dependencies]
 tiny-keccak = "1.4"
-primitive-types = { path = "../primitive-types", version = "0.5", default-features = false }
+primitive-types = { path = "../primitive-types", version = "0.6", default-features = false }
```

Comment:
keccak-hash's version needs to be bumped as well

Reply:
mybad